src/components/app/reducers.js

```diff
@@ -0,0 +1,33 @@
+import Cookies from 'cookies-js';
+
+import { SET_COOKIE, REMOVE_COOKIE } from './actionTypes';
+import { EXPIRATION } from './constants';
+
+const initialState = {
+    loggedIn: false,
+};
+
+export default ( state = { ...initialState }, action) => {
+    switch (action.type) {
+    case SET_COOKIE:    {
+        const { token } = action;
+        Cookies.set(token, { expires: EXPIRATION });
+
+        return {
+            ...state,
+            loggedIn: true,
+        };
+    }
+    case REMOVE_COOKIE: {
+        const { token } = action;
```

Comment:
when you call `removeCookie` are you passing in a token? I told you to save the token on the state and pull it from the state when you want to remove it